index.js

```diff
@@ -66,7 +66,7 @@ try {
 
         const sshConfig = `\nHost key-${sha256}.github.com\n`
                               + `    HostName github.com\n`
-                              + `    IdentityFile ${homeSsh}/key-${sha256}\n`
+                              + `    IdentityFile ~/.ssh/key-${sha256}\n`
```

Comment:
```suggestion
                              + `    IdentityFile %d/.ssh/key-${sha256}\n`
```

this might/should also work